pmd-core/src/main/java/net/sourceforge/pmd/AbstractPropertyDescriptorFactory.java

```diff
@@ -28,30 +30,42 @@
  */
 public abstract class AbstractPropertyDescriptorFactory<T> implements PropertyDescriptorFactory<T> {
 
-    protected static final Map<PropertyDescriptorField, Boolean> CORE_FIELD_TYPES_BY_KEY
-        = CollectionUtil.mapFrom(new PropertyDescriptorField[] {NAME, DESCRIPTION, DEFAULT_VALUE, DELIMITER},
-                                 new Boolean[] {true, true, true, false});
+
+
+
+    static {
+        System.err.println(CORE_FIELD_TYPES_BY_KEY);
```

Comment:
Yeah I ran into an absurd bug and pushed to see if travis reproduced. The bug is that, in the static initializer of PropertyDescriptorUtil, for some reason the FACTORY of IntegerProperty is null. It's uncomprehensible, since it only happens for this specific factory, but the others are declared nearly identically. Moreover, the factory is static final and assigned in its declaration with `new` (which can't return null as per the JLS). You can see the log of the error on the travis log, but you'll need a debugger to see what I mean. I don't know what to do...